### ISSUE TITLE
Use of indiciaData.mapdiv makes assumption that only one map on form.

### ISSUE THREAD

**Issue by Gary-van-Breda** [Indicia-Team/media#4]
Applies to client_helpers as well: primarily impacts map_helper, though a lot of forms use it.
When a form uses 2 maps - e.g. the sectioned transect input transect form and its derivatives (e.g. ukbms) - the indiciaData.mapdiv gets set to the second map initialised. This means things like find me and the tabbed map redraw don't work on the primary (first) map . It is also used in the reporting and associated filters a lot.
I believe its use in its current form needs to be reviewed.

**Comment by johnvanbreda** [comment]
Agreed, though I think it has always been this way. There are a few instances where we reparent the map onto a different container then call map.updateSize if it is needed in 2 places.
